packages/adapter-utils/src/template.ts

```diff
@@ -25,9 +25,27 @@ export type TemplateContainer = {
     fieldName: string
   }
 
+export const compactString = (
+  part: TemplatePart, allParts: TemplatePart[]
+): void => {
+  const tempPart = allParts[allParts.length - 1]
+  if (_.isString(part) && _.isString(tempPart)) {
+    allParts.pop()
+    allParts.push(tempPart.concat(part))
+  } else {
+    allParts.push(part)
+  }
+}
+
+export const createTemplateExpression = (parts: {parts: TemplatePart[] }):TemplateExpression => {
```

Comment:
nits: 
1. maybe `createCompactTemplateExpression`?
```suggestion
export const createTemplateExpression = (parts: { parts: TemplatePart[] }): TemplateExpression => {
```